fabric-client-gametest-api-v1/src/client/java/net/fabricmc/fabric/impl/client/gametest/screenshot/TestScreenshotComparisonOptionsImpl.java

```diff
@@ -89,8 +90,18 @@ public TestScreenshotComparisonOptions withRegion(int x, int y, int width, int h
 		return this;
 	}
 
-	public String getTemplateImagePath() {
-		return this.templateImage.left().orElseThrow();
+	/**
+	 * Gets the path to the template image, relative to the {@code templates} directory, if one was provided.
+	 */
+	public Optional<String> getTemplateImagePath() {
+		return this.templateImage.left();
+	}
+
+	/**
+	 * Gets the path to the template image, relative to the {@code templates} directory, if one was provided.
```

Comment:
update docs to reflect that this method throws (maybe add a throws clause to the method as well(

Reply:
This is an internal method, the docs don't need to be extensive, or present at all, they're only there for future maintainers